sdl-lang/src/main/scala/io/smartdatalake/lab/SmartDataLakeBuilderLab.scala

```diff
@@ -39,3 +39,11 @@ case class SmartDataLakeBuilderLab[T](
   @transient val context: ActionPipelineContext = ConfigToolbox.getDefaultActionPipelineContext(session, registry)
   @transient val data: T = dataObjectCatalogFactory(registry, context)
 }
+
+object SmartDataLakeBuilderLab {
+  /**
+   * Writing into DataObjects using SmartDataLakeBuilderLab is disabled by default, as this should be done through data pipelines and not interactively.
+   * It can be enabled by setting the following variable to true.
+   */
+  var enableWritingDataObjects = false
```

Comment:
OK now I'm a bit confused on how you use this new feature.
I thought the idea was to generate the class files with the CLI.
What is the use case that you satisfy when you set this property to true? You may want to add the answer as documentation in the code

Reply:
This setting is used at Runtime, not for generating the Catalog class file.
We discussed today in the team at a customer. There was a concern that by making it ease to write to a DataObject interactively (e.g. from Polynote), we will loose on governance writing new data should be done by SDLB, e.g. executing Actions. Nevertheless we agreed that it should not be removed completly, but disabled by default...